cypress/integration/home.spec.ts

```diff
@@ -0,0 +1,6 @@
+describe('home page', () => {
+  it('the p tag contains the correct text', () => {
+    cy.visit('http://localhost:3000');
+    cy.get('p').should('exist').contains('Edit <code>src/App.tsx</code> and save to reload');
```

Comment:
I think `.should('exist')` seems a bit redundant as we're asserting `contains` as well.

I have two suggestions here

1. We can keep the assertion simpler (e.g. `cy.get('X').contains('...')`)
2. We have a [convention](https://nimblehq.co/compass/development/code-conventions/javascript/cypress/#element-selector) to use `data-test-id` so that we can use the `findByTestId` method to select the element

Let me know what you think 😄

Reply:
The second suggestion would be better because I think if there are many Html tags that use the same name that would be hard to test because you have to specify the index of the tag e.g. `cy.get('X').eq(0)` so `data-test-id` and `findByTestId` should be a better solution.

Reply:
Fixed on this [commit](https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/30/commits/4006342fb8cd9bc627cb3a1cc80b9fb474d110b5)